src/graph/validator/test/ACLValidatorTest.cpp

```diff
@@ -142,24 +142,6 @@ TEST_F(ACLValidatorTest, Simple) {
     ASSERT_EQ(*changePassword->newPassword(), newPassword);
   }
 
-  // grant role
```

Comment:
Why delete this?

Reply:
Because there is metaclient in this function.

Reply:
This case should keep.